src/server/index.js

```diff
@@ -3,11 +3,11 @@ import { render } from '@jaredpalmer/after';
 import routes from '../app/routes';
 import Document from '../app/components/Document';
 
-const getPublicDirectory = () =>
-  process.env.NODE_ENV === 'production'
+export function getPublicDirectory() {
```

Comment:
We shouldn't export this function just to test it.

It would be better test by mocking these environment variables and then testing whether you can make a [request](https://github.com/bbc/simorgh/pull/272/files#diff-e4ada67d37484d66abff4df36e176558R37) to the [static route](https://github.com/bbc/simorgh/pull/272/files#diff-6e0d62f54b853b53af874f5965d7adf2R28) that is setup from that env variable.